llmcompare/runner/runner.py

```diff
@@ -84,10 +86,50 @@ def single_token_probs(
         result = dict(sorted(result.items(), key=lambda x: x[1], reverse=True))
         return result
 
+    def specific_token_logits(
+        self,
+        messages: list[dict],
+        tokens: list[int] | list[str],
```

Comment:
I think it's better to have two arguments for this.
Main reason: if someone passes
`["123"]`
when they intended to pass
`[123]`
this might be very hard to debug. This happens to me sometimes!